### ISSUE TITLE
Resize large pictures

### ISSUE THREAD

**Issue by lfvperes** [lfvperes/jaguar-bot#2]
The Twitter API documentation states in [this guide](https://developer.twitter.com/en/docs/twitter-api/v1/media/upload-media/uploading-media/media-best-practices), under the section **Image specifications and recommendations**, that the limit for image file sizes is 5MB
- [x] Try to verify the size of the image before downloading, if possible
- [x] Method to resize images
- [ ] Test: Download a large picture, rescale it and tweet it

**Comment by lfvperes** [comment]
Can't check image size before downloading. Will try to do so before posting.

**Comment by lfvperes** [comment]
Apparently some images take up a lot of space for no reason. I had to deal with a image file with size 12 MB, but then I "resized" with a factor of 1 (that is, I didn't really resize) and it came back with size 2 MB (same width, height and quality). The only difference I noticed from input to output was loss of EXIF metadata, which can't be responsible for 10 MB alone. I don't know what the problem was but most likely it was saved as a format different from the original, or an error during download (as all other images have less than 2MB).
